test/server/gameSystems/DrawSystem.spec.js

```diff
@@ -0,0 +1,50 @@
+describe('Drawing cards', function() {
+    integration(function() {
+        describe('When a player draws cards', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['patrolling-vwing', 'mission-briefing'],
+                        deck: ['wampa']
+                    },
+                    player2: {
+                        deck: ['cartel-spacer']
+                    }
+                });
+            });
+
+            it('over the amount in their deck, they should draw the remainder and take damage equal to thrice the difference', function () {
+                this.player1.clickCard(this.missionBriefing);
+                this.player1.clickPrompt('You');
+                expect(this.player1.base.damage).toBe(3);
+                expect(this.player1.hand.length).toBe(2);
+                expect(this.wampa).toBeInLocation('hand');
+                expect(this.player1.deck.length).toBe(0);
+            });
+
+            it('over the amount in their deck, they should draw the remainder and take damage equal to thrice the difference', function () {
```

Comment:
Since the test case name is what gets emitted when the test fails, need to make sure every case has a distinct name so that people can reliably find it in case of failure. Probably just putting "controller" and "opponent" in the names here would be enough